karyon3-core/src/main/java/com/netflix/karyon/governator/AutoBindModule.java

```diff
@@ -0,0 +1,197 @@
+package com.netflix.karyon.governator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import com.google.common.collect.Lists;
+import com.google.common.reflect.ClassPath;
+import com.google.inject.AbstractModule;
+import com.google.inject.Module;
+import com.google.inject.util.Modules;
+import com.netflix.governator.annotations.AutoBindSingleton;
+import com.netflix.karyon.AbstractKaryonModule;
+import com.netflix.karyon.KaryonModule;
+
+/**
+ * Backwards compatibility support for governator's AutoBindSingleton annotation. 
+ * 
+ * To enable,
+ * 
+ * <code>
+ * Karyon.create()
+ *       .apply(new AutoBindModule()
+ *          .includePackages("com.example")
```

Comment:
It may make sense to only support includePackages to avoid confusions. It would be rare where one would include a package then ignore a few nested packages.